lib/js/app/queryCreator/components/TargetProperty/TargetProperty.styles.ts

```diff
@@ -7,3 +9,17 @@ export const Container = styled.div`
 export const PropertyOverflow = styled.div`
   overflow: hidden;
 `;
+
+export const TooltipContent = styled.div`
```

Comment:
There is ~24 references to TooltipContent in this code. I think this is right now a candidate to a separate reusable component. Could You please refactor it to create a reusable single component with tooltip and it's content ?